FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/FreeRTOS_Sockets.c

```diff
@@ -2960,6 +2983,15 @@ void vSocketWakeUpUser( FreeRTOS_Socket_t *pxSocket )
 		}
 		else
 		{
+			if( xIsInputStream != 0 )
```

Comment:
Can we do `if( xIsInputStream != pdFALSE )`? Just for consistency and 0 it technically a magic number.

Reply:
Yes sure, pdFALSE is better.
Also, now I combined the macro calls with existing code:
~~~c
-    if( xIsInputStream != 0 )
+    if( xIsInputStream != pdFALSE )
     {
+        iptraceMEM_STATS_CREATE( tcpRX_STREAM_BUFFER, pxBuffer, uxSize );
         pxSocket->u.xTCP.rxStream = pxBuffer;
     }
     else
     {
+        iptraceMEM_STATS_CREATE( tcpTX_STREAM_BUFFER, pxBuffer, uxSize );
         pxSocket->u.xTCP.txStream = pxBuffer;
     }
~~~

Reply:
Can you have a look at [this](https://github.com/FreeRTOS/FreeRTOS/pull/44#pullrequestreview-383211764)?